src/resolvers.ts

```diff
@@ -1,5 +1,46 @@
+import { getRepository } from 'typeorm';
+import { User } from './entity/user';
+import * as bcrypt from "bcrypt";
+
+const saltRounds = 10;
+
 export default {
   Query: {
     hello: () => 'Hello, world!',
+    user: async (_source, args) => {
+      const user = await getRepository(User).findOne({ id: args.id });
+      if (!user) {
+        console.log('Usuário não encontrado.');
+      }
+      return user;
+    },
+    users: async (_source) => {
+      const response = await getRepository(User).find({
+        select: ['id', 'name', 'email', 'password', 'birthDate'],
+      });
+      return {
+        users: response
+      };
+    },
+  },
+  Mutation: {
+    createUser: async (_source, args) => {
+      const email = await getRepository(User).findOne({
+        where: {
+          email: args.email,
+        },
```

Comment:
Quando só o where é usado, vc pode omitir:

```suggestion
        email: args.email,
```

Reply:
Entendi! Vou ajustar